### ISSUE TITLE
Artipie user experience is completely broken: quick start

### ISSUE THREAD

**Issue by Sammers21** [artipie/artipie#277]
The instructions you need to follow in order to start a simple repository is hard to follow. If someone would try make something simple with artipie he will give up in a few minutes. In comparison with artifactory, which can be started in just one line, artipie has the worst possible user experience ever.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8), @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova) please, pay attention to this issue

**Comment by g4s8** [comment]
@Sammers21 it's not really constructive: how would you suggest improving it?

**Comment by pavlukhin** [comment]
Actually +1 for this issue. A attempted to try out a binary distribution. I used following instructions from https://github.com/artipie/artipie/blob/master/README.md

> The fastest way to start using Artipie is via
[Docker](https://docs.docker.com/get-docker/). First,
create a new directory `artipie` and `repo` sub-directory inside it. Then, put your
YAML config file into the `repo` sub-dir. Make sure that the name of your config file
is the name of repository you are going to host, and its name matches `[a-z0-9_]{3,32}`.
For example `foo.yaml`:

```yaml
repo:
  type: maven
  storage:
    type: fs
    path: /var/artipie
```

>Now, go back to `artipie` and start the container:

```bash
$ docker run -v "$(pwd):/var/artipie" -p 8080:80 artipie/artipie
```

>You should be able to use it with [Maven](https://maven.apache.org/)
at `http://localhost:8080`.

After several trial and error rounds (with exploring source code) I finally entered a web page. I did following for it:
1. Stopped using a root path at `http://localhost:8080` and switched to `http://localhost:8080/test`.
2. Manually created `_permissions.yaml` file.
3. Manually created `_credentials.yml` file with a user `test`.
4. Manually created  `repo/test` folder and moved mentioned in the instruction `foo.yaml` there.

P.S. Have no chance yet to try out working with repository packages (e.g. Maven artifacts).
